internal/utils/aggregator.go

```diff
@@ -40,6 +42,33 @@ func Aggregate_data(df dataframe.DataFrame) dataframe.DataFrame {
 
 	df = df.Mutate(s.Col("X0")).Rename("k8s_object_type", "X0")
 	df = df.Mutate(s.Col("X1")).Rename("k8s_object_name", "X1")
+
+	// filter out only valid workload type
+	df = df.Filter(
```

Comment:
Is it possible to get this out as separate function? I understand its just a call to Filter however the `Aggregate_data` is already lengthy. With that can we have tests then? So looking at those one would understand what is invalid?

Reply:
Added test cases